File: 1304. Find N Unique Integers Sum up to Zero/solution.py
class Solution:
    def sumZero(self, n: int) -> List[int]:
        if n == 1:
            return [0]

        res = []
        sum = 0
        for idx in range(n-1):
            cur = idx + 1
            res.append(cur)
            sum += cur

        res.append(-sum)
        return res

File: 1304. Find N Unique Integers Sum up to Zero/solution.go
func sumZero(n int) []int {
	if n == 1 {
		return []int{0}
	}

	res := make([]int, n)
	sum := 0

	for idx := range n - 1 {
		cur := idx + 1
		res[idx] = cur
		sum += cur
	}

	res[n-1] = -sum
	return res
}
